src/include/gcc.h

```diff
@@ -248,12 +248,14 @@ WT_ATOMIC_FUNC(size, size_t, size_t *vp, size_t v)
     do {                                            \
         __asm__ volatile("dmb ishld" ::: "memory"); \
     } while (0)
-/*
- * This is necessary as we're moving to utilizing acquire and release semantics. The ARM load
- * barrier does provide us with acquire semantics but the store barrier does not. We upgrade it to a
- * full barrier for that reason.
- */
-#define WT_WRITE_BARRIER() WT_FULL_BARRIER();
+/* In order to achieve release semantics we need a both a load and store barrier. */
+#define WT_WRITE_BARRIER() \
```

Comment:
In the comment can we call out that 
- `dmb ishst` gives StoreStore ordering
- `dmb ishld` gives LoadLoad + LoadStore ordering
- In combination this gives us the required StoreStore + LoadStore ordering needed for release semantics
?

It's easy (i.e. me right now) to forget that `ishld` gives the required LoadStore.

Reply:
a171ef475